subtests/docker_cli/events/events.py

```diff
@@ -1,3 +1,5 @@
+#
+# coding: utf-8
```

Comment:
This might be better done as a mass-change all at once and tucked into a single commit by itself, just in case.